docs/frida_mode.md

```diff
@@ -0,0 +1,29 @@
+# Binary-only fuzzing with Frida mode
+
+fuzzuf can perform a bianry-only fuzzing by utilizing [AFL\+\+'s Frida mode](https://github.com/AFLplusplus/AFLplusplus/tree/stable/frida_mode).  
+It can be used for fuzzing a binary built without a compile-time instrumentation. Make sure the executable is dynamically linked as it needs to inject the library with `LD_PRELOAD`.
+
+Currently, there are limitations on using Frida mode on fuzzuf:
+- Only available on x86-64 Linux through CLI.
+- [Scripting](https://github.com/AFLplusplus/AFLplusplus/blob/stable/frida_mode/Scripting.md) is not available
+
+
+## Build instruction
+
+To use Frida mode, specify `-DENABLE_FRIDA_TRACE=1` while generating a CMake's build system.  
+It will clone `AFLplusplus` submodule and build `afl-frida-trace.so` located under `AFLplusplus/frida_mode/` directory.
+
+```console
+$ cd /path/to/fuzzuf
+$ # Build plain (dynamically linked, uninstrumented) `exifutil` executable
+$ make -C docs/resources/exifutil clean all
+$ cmake -B build ... -DENABLE_FRIDA_TRACE=1
```

Comment:
```suggestion
$ cmake -B build . -DENABLE_FRIDA_TRACE=1
```
If the current directory hasn't changed after `cd /path/to/fuzzuf`, then I guess one dot is the right answer.

Reply:
I've put these three dots as a placeholder for other `cmake` options and that was a bad decision. Sorry for making you confused.